interfaces/builtin/network_bind_test.go

```diff
@@ -56,11 +71,7 @@ func (s *NetworkBindInterfaceSuite) TestName(c *C) {
 func (s *NetworkBindInterfaceSuite) TestSanitizeSlot(c *C) {
 	err := s.iface.SanitizeSlot(s.slot)
 	c.Assert(err, IsNil)
-	err = s.iface.SanitizeSlot(&interfaces.Slot{
-		Snap:      "some-snap",
-		Name:      "network-bind",
-		Interface: "network-bind",
-	})
+	err = s.iface.SanitizeSlot(s.otherSnapSlot)
```

Comment:
It would be better to preserve the logic here closer to the original test, with a trivial Slot created in place. Nothing else uses "otherSnapSlot", with a name of "some-snap". The test file was made longer and more obscure with no plus sides.